Lesson14/Gokhan/script.js

```diff
@@ -0,0 +1,29 @@
+// 9. Declare a variable number6 and initialize it to 6. 
+//Declare a variable number7 and initialize it to `++number6` (number6 incremented using prexif ++ operator). Log both values to the console.
+
+const number6 = 6;
+const number7 = ++6;
```

Comment:
Here you need to use a variable, not just a number.
```suggestion
const number7 = ++number6;
```